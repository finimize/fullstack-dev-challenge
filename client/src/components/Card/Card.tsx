import React, { FC } from 'react'
import { Box, Text } from '@chakra-ui/react'

type CardProps = {
    children: React.ReactNode
}

export const Card: FC<CardProps> = ({ children }) => (
    <Box margin='8' backgroundColor='white' borderRadius='8px' padding='8'>
        {children}
    </Box>
)
