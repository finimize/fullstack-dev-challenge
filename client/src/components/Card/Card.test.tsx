import React from 'react'
import { render } from '@testing-library/react'
import { Card } from './Card'

describe('components/Card', () => {
    it('renders the Card Component with a child', () => {
        const { container } = render(
            <Card>
                <div>Hello</div>
            </Card>,
        )
        expect(container).toMatchSnapshot()
    })
})
