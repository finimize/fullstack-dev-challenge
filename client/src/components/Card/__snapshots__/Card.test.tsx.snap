// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Card renders the Card Component with a child 1`] = `
<div>
  <div
    class="css-mqjv2r"
  >
    <div>
      Hello
    </div>
  </div>
</div>
`;
